go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -455,18 +453,10 @@ func (as *ovnAddressSet) destroy() error {
 	return nil
 }
 
-func getIPv4ASName(name string) string {
-	return name + ipv4AddressSetSuffix
-}
-
-func getIPv6ASName(name string) string {
-	return name + ipv6AddressSetSuffix
-}
-
-func getIPv4ASHashedName(name string) string {
-	return hashedAddressSet(name + ipv4AddressSetSuffix)
+func MakeAddressSetName(name string) (string, string) {
+	return name + ipv4AddressSetSuffix, name + ipv6AddressSetSuffix
 }
 
-func getIPv6ASHashedName(name string) string {
-	return hashedAddressSet(name + ipv6AddressSetSuffix)
+func MakeAddressSetHashNames(name string) (string, string) {
+	return hashedAddressSet(name + ipv4AddressSetSuffix), hashedAddressSet(name + ipv6AddressSetSuffix)
```

Comment:
@JacobTanenbaum could we actually call MakeAddressSetName() here instead of re-constructing the names? A bit more consistent.

Reply:
done